app/assets/javascripts/components/common/overview_stats_tabs.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import OverviewStatsTab from './OverviewStats/overview_stats_tab';
+import NamespaceOverviewStats from './OverviewStats/namespace_overview_stats';
+import WikidataOverviewStats from './wikidata_overview_stats';
+
+import ArticleUtils from '../../utils/article_utils';
+
+
+const OverviewStatsTabs = ({ statistics }) => {
+  const [currentTabId, setCurrentTabId] = useState(0);
+
+  const getTabTitle = (ns_id, wiki) => {
```

Comment:
These two functions should probably be in a utils file rather than in this component. Tests for them would also be good.